### DISCUSSION TITLE
Add-PnPMicrosoft365GroupOwner : Adding user and changing role manually shows weird behavior

### DISCUSSION THREAD

**Discussion by KarishmaSahuji** [pnp/powershell#4547] [General]
<div type='discussions-op-text'>

<sup>Originally posted by **KarishmaSahuji** November 11, 2024</sup>
Hi Team,
I am using Add-PnPMicrosoft365GroupOwner command to add owners in my site .
But, after adding owners , if I manually go to my SharePoint site and change the role of the newly added Owner to Member , the user disappears completely. 
Also, I noticed when I try this again and again , the group disappears from Outlook.

So we checked the following:
Case 1:
add user to a group using Add-PnPMicrosoft365GroupOwner command
try to change the role
user is removed

Case 2:
add user from outlook or directly from azure
try to change the role
user stays with the role change

Any help here will be much appreciated.

Thanks and Regards,
Karishma Sahuji
 </div>

**Comment by sympmarc** [comment]
I found this discussion while looking for issues with `Add-PnPMicrosoft365GroupOwner`. I don't see the behavior described here, but I have a question about the result of the cmdlet.

When we add an Owner through the UI, they automagically also become a Member. If we use `Add-PnPMicrosoft365GroupOwner` they are *only* made an Owner, and do not have the Member role. We can certainly use `Add-PnPMicrosoft365GroupMember` to add the same person as a Member, but should the `Add-PnPMicrosoft365GroupOwner` cmdlet do what happens in the UI and also make them a Member?
